frontend/src/components/zero-header.tsx

```diff
@@ -1,16 +1,68 @@
-import React from "react";
+import React, {FunctionComponent, PropsWithChildren, useState} from "react"
+import {Button} from "primereact/button";
+import {Sidebar} from "primereact/sidebar";
+import {css} from "@emotion/react";
+import {To, useNavigate} from "react-router-dom";
 
-export const ZeroHeader = () => (
-    <h1 style={{
-        position: 'absolute',
-        left: 0,
-        padding: '.5em 1em',
-        margin: 0,
-    }}>
-        <img
-            src="https://zenmo.com/wp-content/uploads/elementor/thumbs/zenmo-logo-website-light-grey-square-o1piz2j6llwl7n0xd84ywkivuyf22xei68ewzwrvmc.png"
-            style={{height: '1em', verticalAlign: 'sub'}}/>
-        &nbsp;
-        Zenmo Zero
-    </h1>
-)
\ No newline at end of file
+const sidebarStyle = css({
+    width: '250px',
```

Comment:
Please use rem or em so it scales cleaner if the user has a bigger font size set in OS or browser.